onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -627,6 +629,7 @@ public static ObaRegion getRegionFromBuildFlavor() {
                 BuildConfig.FIXED_REGION_SUPPORTS_OBA_DISCOVERY_APIS,
                 BuildConfig.FIXED_REGION_SUPPORTS_OBA_REALTIME_APIS,
                 BuildConfig.FIXED_REGION_SUPPORTS_SIRI_REALTIME_APIS,
+                BuildConfig.FIXED_REGION_SUPPORTS_EMBEDDEDSOCIAL,
```

Comment:
I believe Embedded Social should be moved tot he last index here, as it's the most recently added database field.